### PR TITLE
Add staking section

### PR DESCRIPTION
**Pull request by jiyoung-an**
Update:
- Add `staking` section

### CODE REVIEW

File: ain-blockchain/staking/staking.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+```
+$ npm install
+```
+
+3. Run the staking script
+   Use the provided [sendStakeTx.js](https://github.com/ainblockchain/ain-blockchain/blob/master/tools/staking/sendStakeTx.js) script to stake your tokens:
+
+```sh
+$ node sendStakeTx.js <BLOCKCHAIN_API_ENDPOINT> <CHAIN_ID> <APP_NAME> <STAKING_AMOUNT> <ACCOUNT_TYPE> [<KEYSTORE_FILE_PATH>]
+$ node sendStakeTx.js https://testnet-api.ainetwork.ai 0 test_app 100 keystore keystore_user.json
```
Comment (jiyoung-an) [inline ain-blockchain/staking/staking.md]: @platfowner does the private_key user need a keystore file to use this script (sendStakeTx.js) ?

File: ain-blockchain/staking/staking.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
+# Staking
+
+## What is staking?
+
+Staking is a way to lock your tokens to help the blockchain network or applications run smoothly. In return, you may earn rewards or contribute to the stability of the system.
+Think of it like putting your money in a savings account: you're helping the system grow, and sometimes you earn interest.
+
+## How does staking work?
```
Comment (jiyoung-an) [inline ain-blockchain/staking/staking.md:8]: @platfowner please double-check here!

File: ain-blockchain/staking/staking.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+- `<BLOCKCHAIN_API_ENDPOINT>`: The endpoint of the blockchain API. (e.g. testnet: 'https://testnet-api.ainetwork.ai', mainnet: 'https://mainnet-api.ainetwork.ai')
+- `<CHAIN_ID>`: The ID of the chain. (0: testnet, 1: mainnet)
+- `<APP_NAME>`: The name of the app.
+- `<STAKING_AMOUNT>`: The amount of tokens to stake.
+- `<ACCOUNT_TYPE>`: The type of the account: private_key, mnemonic, keystore
+- `<KEYSTORE_FILE_PATH>`: The path to the keystore file.
+
+4. Confirm the transaction
+
+## How to decide the staking amount?
```
Comment (jiyoung-an) [inline ain-blockchain/staking/staking.md:120]: @platfowner please double-check here!

File: ain-blockchain/staking/staking.md (hunk 1 above)
Comment (platfowner) [reply ain-blockchain/staking/staking.md]: it supports private_key and mnemonic options as well.
https://github.com/ainblockchain/ain-blockchain/blob/master/tools/staking/sendStakeTx.js#L65

maybe you can describe that as well.

File: ain-blockchain/staking/staking.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+To determine the right staking amount for your app, consider the following key points:
+
+1. Understand resource needs
+
+- Your app requires two budgets:
+  - **Bandwidth Budget:** Calculated as the number of transactions per second (TPS)
```
Comment (platfowner) [inline ain-blockchain/staking/staking.md]: Actually, it's not about tps. it's something like "The number of the DB write operations needed to execute the transaction.

File: ain-blockchain/staking/staking.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+  - **State Budget:** Based on the memory used in the blockchain's state tree (limit: 5GB shared across all apps).
```
Comment (platfowner) [inline ain-blockchain/staking/staking.md]: Could you just remove 5GB part?

File: ain-blockchain/staking/staking.md [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+Staking is a way to lock your tokens to help the blockchain network or applications run smoothly. In return, you may earn rewards or contribute to the stability of the system.
+Think of it like putting your money in a savings account: you're helping the system grow, and sometimes you earn interest.
+
+## How does staking work?
+
+In our system, staking works a bit differently depending on where you stake your tokens:
+
+1. **Staking in the consensus app**
+
+- **What it does:** Your tokens are used to create new blocks on the blockchain.
```
Comment (platfowner) [inline ain-blockchain/staking/staking.md]: How about just mentioning that AIN Blockchain uses PoS consensus algorithm and the staking amount with the 'consensus' app is used for the purpose?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: ain-blockchain/staking/staking.md (hunk 1 above)
Comment (jiyoung-an) [reply ain-blockchain/staking/staking.md]: Thank you!

File: ain-blockchain/staking/staking.md (hunk 4 above)
Comment (jiyoung-an) [reply ain-blockchain/staking/staking.md]: Thank you for review this!

File: ain-blockchain/staking/staking.md (hunk 6 above)
Comment (jiyoung-an) [reply ain-blockchain/staking/staking.md]: done!

File: ain-blockchain/staking/staking.md (hunk 5 above)
Comment (jiyoung-an) [reply ain-blockchain/staking/staking.md]: done!

**Comment by jiyoung-an** [comment]
Thank you for review @platfowner

**Pull request merged**